### DISCUSSION TITLE
Client WS wrapper to handle reconnection

### DISCUSSION THREAD

**Discussion by cmnstmntmn** [uNetworking/uWebSockets.js#116] [Q&A] [answered]
hey, great work!

i'm wondering what you guys are using for that.
i found [reconnecting-websocket
](https://github.com/joewalnes/reconnecting-websocket) but is kind of old.

**Comment by porsager** [comment]
I'm using https://github.com/porsager/pws 😉

**Comment by ghost** [comment, marked as answer]
Holy shit 365 lines of code to reconnect?

Try `ws.onclose = () => {whatever_you_ran_to_connect();}`

**Comment by cmnstmntmn** [comment]
hehe, thank you @porsager.
